src/asset-swapper/swap_quoter.ts

```diff
@@ -87,9 +88,16 @@ export class SwapQuoter {
         const samplerBytecode = _.get(artifacts.ERC20BridgeSampler, 'compilerOutput.evm.deployedBytecode.object');
         // Allow address of the Sampler to be overridden, i.e in Ganache where overrides do not work
         const samplerAddress = (options.samplerOverrides && options.samplerOverrides.to) || SAMPLER_ADDRESS;
+        // Allow the UniV3 MultiQuoter bytecode to be written to a specic address
+        const uniV3MultiQuoterBytecode = _.get(
+            artifacts.UniswapV3MultiQuoter,
+            'compilerOutput.evm.deployedBytecode.object',
+        );
+
         const defaultCodeOverrides = samplerBytecode
             ? {
                   [samplerAddress]: { code: samplerBytecode },
+                  [UNISWAP_V3_MULTIQUOTER_ADDRESS]: { code: uniV3MultiQuoterBytecode },
```

Comment:
nit/optional (maybe a future optimization): attach this on chains where uniswap v3 is deployed (it would a waste on other chains and bloat the request size).